community/cypher/cypher/src/test/scala/org/neo4j/cypher/NodeIndexSeekAcceptanceTest.scala

```diff
@@ -104,6 +102,54 @@ class NodeIndexSeekAcceptanceTest extends ExecutionEngineFunSuite with NewPlanne
     result should (use("NodeIndexSeek") and evaluateTo(List(Map("p" -> null, "placeName" -> null))))
   }
 
+  test("should not use indexes when RHS of property comparison depends on the node searched for (equality)") {
```

Comment:
As in the other PR, the test says 'should not use indexes', but makes no assertion on that.